### PR TITLE
Feature/Implementation-of-employees-CRUD-with-AdminLTE-v3.0

### PR DESCRIPTION
**Pull request by JoharAlam**

### CODE REVIEW

File: app/Http/Controllers/EmployeeController.php [hunk 1]
```diff
@@ -109,22 +112,19 @@ public function create()
...
-            'contact_no'     => 'required|unique:employees|size:11',
-            'gender'         => 'required',
-            'picture'        => 'max:1000',
-
-            // 'cnic' => 'size:13',
+            'contact_no'     => 'required|size:11|unique:employees,contact_no',
+            'picture'        => 'image|mimes:jpg,png,jpeg,gif,svg|max:1000',
         ]);
 
         if (! strstr(strtolower($request->official_email), 'glowlogix.com')) {
```
Comment (awaidanjum) [inline app/Http/Controllers/EmployeeController.php:130]: @JoharAlam 
We are making it open source so **"glowlogix.com"** cant be hardcoded, So make it dynamic.

File: resources/views/admin/employees/index.blade.php [hunk 2]
```diff
@@ -1,112 +1,165 @@
...
-                    </tr>
-                    </thead>
-                    <tbody>
-                    @foreach($employees as $employee)
-                    <tr>
-                        <td>{{$employee->firstname}} {{$employee->lastname}}</td>
-                        <td>{{$employee->official_email}}</td>
-                        <td>{{$employee->contact_no}}</td>
-                        <td>{{ ucfirst(trans($employee->designation))}}</td>
-                        {{--<td>{{isset($designations[$employee->designation]) ? $designations[$employee->designation] : ''}}</td>--}}
```
Comment (awaidanjum) [inline resources/views/admin/employees/index.blade.php:46]: @JoharAlam 
Please remove commented code like this.

File: app/Http/Controllers/EmployeeController.php (hunk 1 above)
Comment (JoharAlam) [reply app/Http/Controllers/EmployeeController.php:130]: @awaidanjum I will fix it

File: resources/views/admin/employees/index.blade.php (hunk 2 above)
Comment (JoharAlam) [reply resources/views/admin/employees/index.blade.php:46]: @awaidanjum I will remove it

File: resources/views/admin/employees/edit.blade.php [hunk 3]
```diff
@@ -1,520 +1,672 @@
...
+          <li class="breadcrumb-item active">Edit</li>
+        </ol>
+      </div>
+    </div>
+  </div>
+</div>
+<!-- Breadcrumbs End -->
+
+<!-- Error Message Section Start -->
+@if($errors->any())
```
[the remark this answers is not in the document]
Comment (JoharAlam) [reply resources/views/admin/employees/edit.blade.php:24]: @awaidanjum I will make partial for errors or success massages

**Comment by JoharAlam** [comment]
@faizan-glowlogix bhai Please Review

**Pull request merged**